snapshot.go

```diff
@@ -315,11 +220,20 @@ func getAttemptDownloadClosure(isBoot bool, snapshot *common.Snapshot, uri strin
 
 func processSnapshotServerFileResponse(dbId string, body io.Reader, snapshot *common.Snapshot) error {
 	dbPath := data.DBPath("common/" + dbId)
+	dbDir := dbPath[0 : len(dbPath)-7]
```

Comment:
Consider using config or constants so that as and when dbId length change it is easy to update at one place.

Reply:
I agree. I'll update it with a constant. 
I don't know who was the author of this `len(dbPath)-7` code, but I agree the `7` is impairing  readability. It's in fact the length of the word "/sqlite".